Source/BoundaryConditions/WarpXFieldBoundaries.cpp

```diff
@@ -30,5 +41,20 @@ void WarpX::ApplyBfieldBoundary (const int lev, PatchType patch_type)
             PEC::ApplyPECtoBfield( Bfield_cp[lev], lev, patch_type);
         }
     }
+
```

Comment:
Could you add a comment:
```suggestion

    // Silver-Mueller boundaries are only applied on the first half-push of B
    // This is because the formula used for Silver-Mueller assumes that 
    // E and B are staggered in time, which is only true after the first half-push
```